README.md

```diff
@@ -34,3 +34,11 @@ getURLFor('canary').then((url) => {
   // use the URL here 
 });
 ```
+
+### Proxy
+
+Proxies are supported through `https_proxy`, `http_proxy`, and `no_proxy` environment variables:
```

Comment:
Let's have these in the README as both `SCREAMING_SNAKE_CASE` and lower case, or otherwise make it clearer that we are operating insensitively.